pages/accounts/profile.py

```diff
@@ -30,30 +30,30 @@ def name(self):
     @property
     def username(self):
         """Username field."""
-        return self.User(self)
+        return self.Username(self)
 
     @property
     def emails(self):
         """Email fields."""
-        return self.Email(self)
+        return self.Emails(self)
 
     @property
     def login_method(self):
         """Options for logging in."""
-        return self.LoginOption(self)
+        return self.LoginOptions(self)
 
     def log_out(self):
         """Log the user out."""
         self.find_element(*self._log_out_locator).click()
         sleep(1)
-        return home.AccountsHome(self.driver)
+        return home.Home(self.driver)
```

Comment:
This will probably give a syntax error and should be `home.AccountsHome`; if AccountsHome throws an exception, let me know.